src/backend/setup.cfg

```diff
@@ -25,10 +25,12 @@ classifiers =
 [options]
 install_requires =
     arrow==1.0.3
-    django==3.1.7
+    django==3.2.0
     django-configurations==2.2
     django-cors-headers==3.7.0
     django-countries==7.1
+    django-marion==0.2.0
+    django-marion-howard
```

Comment:
Do not forgot to pin `django-marion-howard`

Reply:
Yes I finish my PR first for a new release.